OpenCL_Ext.txt

```diff
@@ -84,6 +84,8 @@ include::ext/cl_khr_extended_versioning.asciidoc[]
 
 include::ext/cl_khr_subgroup_extensions.asciidoc[]
 
+include::ext/cl_khr_pci_bus_info.asciidoc[]
```

Comment:
This comment says add new extensions here, but that bumps "Extensions to the OpenCL SPIR-V Environment" to chapter 40.

Not sure that is actually a problem.

A separate discussion suggested that perhaps that chapter should be an appendix instead to avoid this bumping problem.